code/game/cas_manager/datums/cas_fire_envelope.dm

```diff
@@ -259,7 +259,26 @@
 		mission_error = "Target is off bounds or obstructed."
 		return
 	change_current_loc(target_turf)
-	playsound(target_turf, soundeffect, 70, TRUE, 50)
+	playsound(source = target_turf, soundin = soundeffect, vol = 70, vary = TRUE, sound_range = 50, falloff = 8)
+
+	var/relative_dir
+	for(var/mob/mob in range(15, target_turf))
+		if(get_turf(mob) == target_turf)
+			relative_dir = 0
+		else
+			relative_dir = Get_Compass_Dir(mob, target_turf)
+
+		var/ds_identifier = "LARGE BIRD"
```

Comment:
**(1/2)**
I know it's the same as on the other CAS messages, but this could be neatened a bit by moving some of the proc calls around:
```suggestion
		else
			relative_dir = Get_Compass_Dir(mob, target_turf)
		relative_dir = uppertext(dir2text(relative_dir))

		var/ds_identifier = "LARGE BIRD"
```